src/technologies/a.json

```diff
@@ -1537,16 +1537,34 @@
     "scripts": "aem-(?:GridColumn|apps/)",
     "website": "https://www.adobe.com/marketing/experience-manager.html"
   },
-  "Adobe Experience Manager Franklin": {
+  "Adobe Experience Manager with Edge Delivery Services preview": {
     "cats": [
       1
     ],
     "cpe": "cpe:2.3:a:adobe:experience_manager:*:*:*:*:*:*:*:*",
-    "description": "Adobe Experience Manager Franklin, also known as Project Helix or Composability, is a new way to publish AEM pages using Google Drive or Microsoft Office via Sharepoint. Instead of components, Franklin uses blocks to build pages. Blocks are pieces of a document that will be transformed into web page content.",
-    "icon": "Adobe Experience Manager Franklin.svg",
-    "excludes": "Adobe Experience Manager",
-    "scriptSrc": "^.+/scripts/lib-franklin\\.js$",
-    "website": "https://www.hlx.live"
+    "description": "Edge Delivery Services in Adobe Experience Manager Sites as a Cloud Service (Preview)",
```

Comment:
Can you explain what `(Preview)` means? Is this a Beta project? Or like a staging version of a site?

Reply:
These are indeed the preview URLs for a specific branch of a website implementation (non-prod deployments).

They are most of the time non-indexed, but in some rare occasions customers might want to leave them accessible. These URLs should be ignored.

Would you have an alternative way to exclude them from the crawling?

Reply:
Ah I see. No can't think of a better way to exclude them. However not sure they'd get enough traffic to really be included here? And if they do, then arguable they should be measured.

So not sure it's worth excluding them either way?

And if anything this will highlight those sites and make them easier to find. Plus there will be people like me confused what exactly "preview" means :-)

I say remove this.

Reply:
Gotcha. Actually, we had a couple of cases which would get enough traffic but for now I followed your feedback and removed the entry. I was also not super happy to have a Preview entry that could be visible in the report.